builder.json

```diff
@@ -6,5 +6,9 @@
     ],
     "downstream": [
         { "name": "aws-c-auth" }
+    ],
+    "test": [
+        ["ctest", ".", "--output-on-failure"],
+        ["{python}", "integration-testing/http_client_test.py", "{install_dir}/bin/elasticurl{exe}"]
```

Comment:
does the path need to go up one level here?

Reply:
Yes, good catch
using "{source_dir}/integration-testing/..." now